src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Scanner;
```

Comment:
Firstly, i think you can have more commits to better show your updates and improve on your commit name/message